src/main/java/net/minecrell/terminalconsole/HighlightErrorConverter.java

```diff
@@ -62,17 +62,23 @@
     private static final String ANSI_RESET = "\u001B[m";
     private static final String ANSI_ERROR = "\u001B[31;1m"; // Bold Red
     private static final String ANSI_WARN = "\u001B[33;1m"; // Bold Yellow
+    private static final String ANSI_INFO = "\u001B[32m"; // Blue
```

Comment:
Considering that TCA is used by some Minecraft server modifications, then turning INFO level blue by default is not a great idea imho

Reply:
First off, coloring below log level warn is disabled by default, you need to set a flag to enable it. Also, this is designed to mimic the log4j highlighter, so I have choosen the same colors as log4j

Reply:
I'm aware that one needs to set a flag to enable it.